core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -792,4 +795,9 @@ public static String getParentPath(String dataFilePath) {
       return dataFilePath;
     }
   }
+
+  public static String getTrashFolderPath(String carbonTablePath) {
+    return carbonTablePath + CarbonCommonConstants.FILE_SEPARATOR + CarbonTablePath
+      .TRASH_DIR;
```

Comment:
just use constant, as its in same class, remove `CarbonTablePath`

Reply:
done